src/Components/GenreSelector/GenreSelector.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { screen, render, fireEvent } from '@testing-library/react';
+import { GenreSelector } from './GenreSelector';
+
+describe('Test GenreSelector Component', () => {
+
+    const genreList = [
+        { id: 0, name: 'all' },
+        { id: 1, name: 'drama' },
+        { id: 2, name: 'terror' },
+        { id: 3, name: 'comedy' },
+        { id: 4, name: 'documentary' },
+        { id: 5, name: 'crime' },
+    ];
+
+    const selectedGenre = 0;
+
+    const onSelectGenre = (e) => {
+        console.log(e);
+    }
+
+    test('renders all genres passedin props', () => {
```

Comment:
Might be interesting to add a test where the list is empty 🤔 
```
test('render message when list is empty');
```
or something like that